test/parallel/test-tls-delayed-attach.js

```diff
@@ -37,30 +39,30 @@ const options = {
   cert: fixtures.readKey('agent1-cert.pem')
 };
 
-const server = net.createServer(function(c) {
+const server = net.createServer(common.mustCall((c) => {
   setTimeout(function() {
     const s = new tls.TLSSocket(c, {
       isServer: true,
       secureContext: tls.createSecureContext(options)
     });
 
-    s.on('data', function(chunk) {
+    s.on('data', common.mustCall((chunk) => {
```

Comment:
This should not be wrapped in `common.mustCall()`, I don't think. It is theoretically possible for a `data` events to be called more than once (although perhaps not without the underlying implementation changing).

Reply:
Perhaps `common.mustCallAtLeast()`?

Reply:
I wouldn't use `common.mustCallAtLeast()` because it will never fail. If the callback never runs, the `process.on('exit')` will fail first. So adding `common.mustCallAtLeast()` will only add cognitive overhead when trying to understand the test.

Reply:
Should the `on('exit')` callback be `common.mustCall()` then?

Reply:
so should I remove`common.mustCall` ?

Reply:
@richardlau  `on('exit')` its giving error

Reply:
> Should the on('exit') callback be common.mustCall() then?

No, because `common.mustCall()` does it's checks via `.on('exit')` so you can't use `common.mustCall()` on `exit` handlers. I think the thing to do is not rely on `common.mustCall()` for this. (I do think we tend to overuse it and should only use it where needed, but I may be in the minority on that.)

Reply:
> so should I removecommon.mustCall ?

IMO, yes.

Reply:
changes made :)
thank you for feedback